pydata_google_auth/auth.py

```diff
@@ -263,7 +301,10 @@ def get_user_credentials(
             if use_local_webserver:
                 credentials = _webserver.run_local_server(app_flow)
             else:
-                credentials = app_flow.run_console()
+                flow.InstalledAppFlow.run_webapp = run_webapp
+                credentials = app_flow.run_webapp(redirect_uri=redirect_uri)
```

Comment:
As far as I can tell, we only use public methods in `run_webapp`. I wonder if monkeypatching is actually necessary?

```suggestion
                credentials = run_webapp(app_flow, redirect_uri=redirect_uri)
```

Reply:
Done.